src/services/experimental-features.test.js

```diff
@@ -0,0 +1,46 @@
+import { ExperimentalFeatures } from './experimental-features';
+
+describe('experimental features', () => {
+  const experimentalFeaturesData = {
+    feature1: true,
+    feature2: true,
+    feature3: {
+      apikey: 'myApiKey',
+    },
+  };
+  localStorage.setItem('dopplerExpermiental', JSON.stringify(experimentalFeaturesData));
+  const experimentalFeatures = new ExperimentalFeatures(localStorage);
+
+  it('should validate if a feature is enabled', () => {
+    // Arrange
```

Comment:
What do you think about preparing the scenario of each test in place of sharing information?

Reply:
Why would I want that? I would be repeating code right? Maybe I'm missing something...

Reply:
In order to read the test as a little story:

1. You arrange the initial state, the scenario.
2. You execute an action.
3. You verify the resulting state.

You want a clear and clean initial scenario to be sure about what is the test doing.

Of course, you could use `beforeEach` to set up a common scenario, but in general, it is not a good idea, because when the scenario is useful for more than a test, probably is not so clean as each test need.

Reply:
When setting up the scenario is difficult, you could use helpers to reuse code.

Reply:
I'll try to fine tune each scenario and see what I get.